tests/components/deconz/test_alarm_control_panel.py

```diff
@@ -103,11 +94,17 @@ async def test_alarm_control_panel(
                 "type": "ZHAAncillaryControl",
                 "uniqueid": "00:00:00:00:00:00:00:00-00",
             }
-        },
-    }
-    with patch.dict(DECONZ_WEB_REQUEST, data):
-        config_entry = await setup_deconz_integration(hass, aioclient_mock)
-
+        }
+    ],
+)
+async def test_alarm_control_panel(
+    hass: HomeAssistant,
+    aioclient_mock: AiohttpClientMocker,
+    config_entry_setup: ConfigEntry,
+    mock_put_request: Callable[[str, str], AiohttpClientMocker],
+    mock_deconz_websocket,
```

Comment:
If order doesn't matter here you could use `@pytest.mark.usefixtures` for `mock_deconz_websocket`

Reply:
Actually it looks like you call it, so would be nice to type it, but it looks like it only uses kwargs?

Reply:
My plan is to type it in its own PR to keep these other PRs to keep them as small as possible